docs/docs/quickstart.md

```diff
@@ -32,20 +32,19 @@ Flagsmith manages Flags with Projects, so let's create one now:
 
 Flagsmith organises Projects into separate Environments. When you create a Project, Flagsmith automatically creates
 `Development` and `Production` Environments. We will come to these Environments later. Let's go ahead and create our
-first Flag. This flag is simply going to control whether a button shows on our super simple web page.
+first Flag. This flag will control whether a button shows on our web page.
 
 <div style={{textAlign: 'center'}}><img width="75%" src="/img/quickstart/demo_create_3.png"/></div>
 
 Flags within Flagsmith are a combination of both: A Boolean value - the `Flag State` and then optionally: A
-String/Integer/Float value - the `Flag Value`. For now, we're just going to use the `Boolean` value of the flag to
-control whether the button shows. Let's go ahead and create a flag called `show_demo_button`. We're going to leave it as
-Disabled by default:
+String/Integer/Float value - the `Flag Value`. For now, we're going to use the `Boolean` value of the flag to control
```

Comment:
Feels like this changes the meaning of the sentence. Perhaps we should change `just` to `only` instead? 

```suggestion
String/Integer/Float value - the `Flag Value`. For now, we're only going to use the `Boolean` value of the flag to control
```